### ISSUE TITLE
BUG: when starting 2 locals in parallel, they start 2 managers

### ISSUE THREAD

**Issue by hagai-lvi** [tomleibo/distributedSystems#59]
When starting 2 locals in parallel, they start 2 managers. is there any way to prevent this?
@tomleibo what do you think?

**Comment by hagai-lvi** [comment]
Another possible scenario:
- 2 running locals, single manager
- manager crashes
- each local starts a new manager
- 2 managers

**Comment by tomleibo** [comment]
When starting the local machine: local sends a new task command to the manager, why should the outcome be a new manager?
We should be able to run the local separately, without creating a manager through the aws cli.
What do you think? @hagai-lvi

**Comment by hagai-lvi** [comment]
Not sure I understand what you mean.
What I meant: When I tried a mini stress test - starting 5 locals in parallel, I've noticed that more than 1 manager starts. and this of course causes wrong behaviour.
Let's talk about it when we meet
